lib/task_listener.js

```diff
@@ -28,6 +25,7 @@ export default class TaskListener extends EventEmitter {
     this.runningTasks = [];
     this.taskQueue = new TaskQueue(this.runtime);
     this.taskPollInterval = this.runtime.taskQueue.pollInterval;
+    this.timestamp = Date.now();
```

Comment:
nit: I probably would name this something more descriptive as to what you're trying to use it for (such as listenerStartTime)